v23.1/delete-data.md

```diff
@@ -220,7 +220,7 @@ with conn.cursor() as cur:
 
 ## Performance considerations
 
-Because of the way CockroachDB works under the hood, deleting data from the database does not immediately reduce disk usage.  Instead, records are marked as "deleted" and processed asynchronously by a background garbage collection process.  This process runs every 25 hours by default to allow sufficient time for running [backups](take-full-and-incremental-backups.html) and running [time travel queries using `AS OF SYSTEM TIME`](as-of-system-time.html).  The garbage collection interval is controlled by the [`gc.ttlseconds`](configure-replication-zones.html#replication-zone-variables) setting.
+Because of the way CockroachDB works under the hood, deleting data from the database does not immediately reduce disk usage.  Instead, records are marked as "deleted" and processed asynchronously by a background garbage collection process.  This process runs [at a specified interval](configure-replication-zones.html#gc-ttlseconds) to allow sufficient time for running [backups](take-full-and-incremental-backups.html) and running [time travel queries using `AS OF SYSTEM TIME`](as-of-system-time.html).  The garbage collection interval is controlled by the [`gc.ttlseconds`](configure-replication-zones.html#gc-ttlseconds) setting.
```

Comment:
I think this was+is incorrect. The garbage collection process is running continuously, not every 25hrs or some controllable interval. It's more that data outside the configured window is _eligible_ to get garbage collected, but we don't go look for it every 25hrs.